packages/react/src/components/OverflowMenu/OverflowMenu.test.jsx

```diff
@@ -13,4 +14,19 @@ describe('OverflowMenu', () => {
 
     expect(screen.getByTestId('overflow_menu')).toBeDefined();
   });
+
+  it('should call the default getMenuOffset from carbon when none given', () => {
```

Comment:
Is that really what we are testing here?

Reply:
Yeah, for some reason after this upgrade our coverage isn't being met on this file--specifically that the carbon getMenuOffset branch wasn't called. I don't know why it changed, but I added this test for coverage to make sure it doesn't blow up when menuOffset is null. 🤷

Reply:
nit: I was just thinking that we aren't actually testing that the carbon getMenuOffset is being called, we are testing that our component handles menuOffset={null} and the onClick is still called